### PR TITLE
Implementation data storage

### PR DESCRIPTION
**Pull request by MHryhoriev**

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""
+This module contains constants used throughout the application.
+"""
+
```
Comment (AlexnderDoroshenko) [inline src/constants.py:4]: Hi @MHryhoriev !
Can we add the user directory here as well?

Examples:
from pathlib import Path
path = Path.home().joinpath( 'Documents', 'mysql_access' )
or
path_string = '~\Documents\mysql_access'
path = Path(path_string).expanduser()

File: src/managers/contact_manager.py [hunk 2]
```diff
@@ -20,28 +20,49 @@
 """
 
 import re
+import os
```
Comment (AlexnderDoroshenko) [inline src/managers/contact_manager.py:23]: Hi @MHryhoriev !
Can we remove unused import ?

**Review by AlexnderDoroshenko — APPROVED** [review]
Minor remarks.
LGTM.

File: src/managers/contact_manager.py (hunk 2 above)
Comment (MHryhoriev) [reply src/managers/contact_manager.py:23]: I may have missed this point, but in fact, such cases will be removed later, when we complete the main development and check the project for such nuances.

File: src/constants.py (hunk 1 above)
Comment (MHryhoriev) [reply src/constants.py:4]: You mean dynamically and cross-platform create a path to our data files?
Something like that:
CONTACT_DATA_FILE_PATH = Path.home().joinpath('data', 'contacts_data.json')
NOTE_DATA_FILE_PATH = Path.home().joinpath('data', 'notes_data.json')

Or do you mean some other user directory?

Comment (AlexnderDoroshenko) [reply src/constants.py:4]: I may be confused, but I think I read somewhere that the files should be stored in the user's directory.
Anyway it's not bad approach to use users dir and create something there.
And of course will be better to do it in cross platform way.

Comment (MHryhoriev) [reply src/constants.py:4]: Let's do it in the following way:
I'll make this code dynamic and cross-platform and consult with our mentor about the user directory thing.
And this is the merging into main

**Pull request merged**